### ISSUE TITLE
cant implement imageCarousel with for loop

### ISSUE THREAD

**Issue by afifmakarim** [line/line-bot-sdk-php#118]
## My code below just showing 1 images from image url json array
```
for($i=0; $i<count($media['nodes']); $i++) {
	if(($i+1)<=10){
	$ImageCarouselTemplateBuilder = new \LINE\LINEBot\MessageBuilder\TemplateBuilder\ImageCarouselTemplateBuilder([
		new \LINE\LINEBot\MessageBuilder\TemplateBuilder\ImageCarouselColumnTemplateBuilder("{$media['nodes'][$i]['thumbnail_src']}",
		new \LINE\LINEBot\TemplateActionBuilder\UriTemplateActionBuilder('Buka Browser',"{$media['nodes'][$i]['thumbnail_src']}")),
		]);
		$templateMessage = new \LINE\LINEBot\MessageBuilder\TemplateMessageBuilder('Instagram',$ImageCarouselTemplateBuilder);
		$result = $bot->replyMessage($event['replyToken'], $templateMessage);
		return $result->getHTTPStatus() . ' ' . $result->getRawBody();
		}
	}
```

> i have json array value of image url and want to implement my code with for loop to get all the images to show in  imageCarousel

**Comment by tokuhirom** [comment]
You can't call the replyMessage method multiple times with the same reply token.

**Comment by afifmakarim** [comment]
@tokuhirom is there any suggestions for my problem? sorry iam beginner to this

**Comment by lintangtimur** [comment]
image link must be https

**Comment by tokuhirom** [comment]
@afifmakarim You can send 5 messages by 1 replyMessage API.
https://developers.line.me/en/docs/messaging-api/reference/#send-reply-message

Then, you can send 5  messages by 1 replyToken.

Example code:
```
                $builder = new MultiMessageBuilder();
                $builder->add(new TextMessageBuilder("hello1"));
                $builder->add(new TextMessageBuilder("hello2"));
                $builder->add(new TextMessageBuilder("hello3"));
                $builder->add(new TextMessageBuilder("hello4"));
                $builder->add(new TextMessageBuilder("hello5"));
                $resp = $bot->replyMessage($event->getReplyToken(), $builder);
```

**Comment by afifmakarim** [comment]
@tokuhirom It is not like that... what i mean is for example:
`$imageJsonArray = array($image[0]['picture'], $image[1]['picture'], $image[2]['picture']);`
and i need to iterate this imageJsonArray into ImageCarouselTemplateBuilder until it show 3 Image Carousel.

**Comment by afifmakarim** [comment]
never mind. solved this problem already.
```
$angka = count($media['nodes']); //variable for count of JSON Array
for($i=0;$i<$angka; $i++){ //for loop until $i fullfilled 
if(($i+1)<=8){ // limit for loop item by 8
$imageUrl = $media['nodes'][$i]['thumbnail_src'];
$actions = new \LINE\LINEBot\TemplateActionBuilder\UriTemplateActionBuilder("open in browser","{$imageUrl}"); 
$column = new \LINE\LINEBot\MessageBuilder\TemplateBuilder\ImageCarouselColumnTemplateBuilder($imageUrl , $actions);
$columns[] = $column; // save imageCarouselColumnBuilder as array
}
}
$carousel = new \LINE\LINEBot\MessageBuilder\TemplateBuilder\ImageCarouselTemplateBuilder($columns);
$templateMessage = new \LINE\LINEBot\MessageBuilder\TemplateMessageBuilder('template name',$carousel);
$result = $bot->replyMessage($event['replyToken'], $templateMessage);
return $result->getHTTPStatus() . ' ' . $result->getRawBody();
```

**Comment by afifmakarim** [comment]
Closes #118
